nixos/modules/services/hardware/bluetooth.nix

```
@@ -146,10 +143,42 @@ in
           {
             wantedBy = [ "bluetooth.target" ];
             aliases = [ "dbus-org.bluez.service" ];
-            serviceConfig.ExecStart = [
-              ""
-              "${package}/libexec/bluetooth/bluetoothd ${escapeShellArgs args}"
-            ];
+            serviceConfig = {
+              ExecStart = [
+                ""
+                "${package}/libexec/bluetooth/bluetoothd ${escapeShellArgs args}"
```

While we're here, I think this should be using `escapeSystemdExecArgs`?

```suggestion
                (escapeSystemdExecArgs ([ "${package}/libexec/bluetooth/bluetoothd" ] ++ args))
```